### ISSUE TITLE
Frio theme:  

### ISSUE THREAD

**Issue by blog2read** [friendica/friendica#2922]
The almost top line with "Event Reminders (x)" seems not to be well formed in CSS - something wrong or missing there.

![CSS](https://i.imgur.com/ygdSzoj.png)

**Comment by rabuzarus** [comment]
Yes I know. This is something that also annoys me. At the moment I don't have a good implementation concept.
Personally I don't want to have it in the stream stream section and I want to put it at another place but at the moment I don't have a real good idea where.
Input and ideas are welcome

BTW:
There are many templates which are missing in frio. It is because a lack of time or because I don't have a good usability vision for it (which is also beautiful). So you will find many things which aren't implemented. And I am really sorry that  I will have not that much time in the next 8 months to start big projects (E.g. the missing template for single photo view really annoys me. But also the smaller parts like events reminder or the user group editing page and much more).

Just for reference,, there is also a open task thread for frio in the bug tracker
